### ISSUE TITLE
Roll 3.4.1?

### ISSUE THREAD

**Issue by chastell** [troessner/reek#717]
Between #710, #713 and #716 (all ‘reek blows up’ fixes) I think it’s time for 3.4.1 – what do you think?

Once #716 is reviewed + merged I would love to roll 3.4.1 – I never did that so I wouldn’t mind seeing how it’s done. :)

**Comment by troessner** [comment]
>> Once #716 is reviewed + merged I would love to roll 3.4.1

Will review this bad boy later and then let's fire away!

>>  – I never did that so I wouldn’t mind seeing how it’s done. :)

Simple: 

* You create a PR like #706 
* After that one is merged you do a pull on master
* And then you fire:

````
rake release
```

This will create a version tag and then push that to rubygems. Since I added you as contributor to rubygems this should just .....work ;)

**Comment by mvz** [comment]
#716 is merged, so :shipit:
